### DISCUSSION TITLE
How do you make a for loop??

### DISCUSSION THREAD

**Discussion by velllu** [elkowar/eww#777] [General]
How can I use a for loop to display 10 buttons, with a label number

**Comment by trueNAHO** [comment]
Related documentation, discussions, and issues:

- [Generating a list of widgets from JSON using for](https://elkowar.github.io/eww/configuration.html?highlight=json#generating-a-list-of-widgets-from-json-using-for)
- https://github.com/elkowar/eww/discussions/442
- https://github.com/elkowar/eww/discussions/656
- https://github.com/elkowar/eww/issues/686
- https://github.com/elkowar/eww/issues/582
